lib/index.js

```diff
@@ -118,6 +118,8 @@ MapboxGeocoder.prototype = {
       return '<div class="mapboxgl-ctrl-geocoder--suggestion"><div class="mapboxgl-ctrl-geocoder--suggestion-title">' + placeName[0]+ '</div><div class="mapboxgl-ctrl-geocoder--suggestion-address">' + placeName.splice(1, placeName.length).join(',') + '</div></div>';
     }
   },
+  
+  _headers: {},
```

Comment:
nit; empty object is "truthy", maybe worth to put `null` as initial value?

Reply:
I think this is ok for how we are using it.